mava/logger.py

```diff
@@ -102,19 +115,52 @@ def log(
                 f"Entropy {float(np.mean(entropy)):.3f}"
             )
             logger.console_logger.info(
-                f"{Fore.MAGENTA}{Style.BRIGHT}TRAINER: {log_string}{Style.RESET_ALL}"
+                f"{Fore.MAGENTA}{Style.BRIGHT}TRAINER{suffix}: {log_string}{Style.RESET_ALL}"
             )
         else:
             logger.console_logger.info(
-                f"{Fore.GREEN}{Style.BRIGHT}EVALUATOR: {log_string}{Style.RESET_ALL}"
+                f"{Fore.GREEN}{Style.BRIGHT}EVALUATOR{suffix}: {log_string}{Style.RESET_ALL}"
             )
 
         return float(np.mean(episodes_return))
 
-    return log
+    def save_json_by_seed() -> None:
+        """Split the current json file by seed into separate files."""
+        # Make sure the full values are recorded in the JSON file.
+        time.sleep(10)
+        # Load JSON data from a file
+        input_file_path = f"{_run.observers[0].dir}/metrics.json"
+        with open(input_file_path, "r") as input_file:
+            data = json.load(input_file)
+
+        # Splitting the data and creating separate JSON files for each seed
+        seed_data: dict = {}
+        for key, value in data.items():
+            metric_name_parts = key.split("_seed_")
+            if len(metric_name_parts) == 2:
+                seed_num = metric_name_parts[1]
+                if seed_num not in seed_data:
+                    seed_data[seed_num] = {}
+                metric_name = metric_name_parts[0]
+                seed_data[seed_num][metric_name] = value
+
+        # Create separate JSON files for each seed's metrics
+        for seed_num, metrics in seed_data.items():
+            output_file_path = f"{_run.observers[0].dir}/metrics_seed_{seed_num}.json"
+            with open(output_file_path, "w") as output_file:
+                json.dump(metrics, output_file, indent=2)
+
+    def stop_logger() -> None:
+        """Stop the logger and save the metrics for different seeds."""
+        if logger.use_neptune:
+            logger.neptune_logger.stop()
+        if (len(config["seeds"]) > 1) and config["split_json_by_seed"]:
+            save_json_by_seed()
```

Comment:
```suggestion
            # Make sure the full values are recorded in the JSON file.
            time.sleep(10)
            save_json_by_seed()
```

Reply:
oh you mean making the time sleep after this condition yeah sure I will update it